src/core/account_address.ts

```diff
@@ -246,8 +246,8 @@ export class AccountAddress extends Serializable {
       } else if (args.input.length !== 3) {
         // 0x + one hex char is the only valid SHORT form for special addresses.
         throw new ParsingError(
-          `The given hex string ${args.input} is a special address not in LONG form, 
-          it must be 0x0 to 0xf without padding zeroes.`,
+            // eslint-disable-next-line max-len
+          `The given hex string ${args.input} is a special address not in LONG form, it must be 0x0 to 0xf without padding zeroes.`,
```

Comment:
we shouldn't need to disable this line. local formatter should know and add a break line  like https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/core/account_address.ts#L249

Reply:
Unfortunately the way that this written before, it did not.  The new line won't handle the same way see:
The code gave this error, where there were extra spaces from the padding.  Otherwise, you'd have to put a `+` and a newline since it can't differentiate between spaces at the beginning and the end of the line.

```
The given hex string 0x000 is a special address not in LONG form, it must be 0x0 to 0xf without padding zeroes.
    + The given hex string 0x000 is a special address not in LONG form, 
    +           it must be 0x0 to 0xf without padding zeroes.

```

Reply:
not completely understand the problem, but lets deal with it now